backend/database/migrations/2022_12_07_155622_update_interviews_table.php

```diff
@@ -14,7 +14,7 @@
     public function up()
     {
         Schema::table('interviews', function (Blueprint $table) {
-            $table->dropColumn('company_name');
+            // $table->dropColumn('company_name');
```

Comment:
why this? pls I need explanation here.... 
And instead of editing a migration file, pls use the update migration method

Reply:
This migration was already there and was causing the migrations to fail because a non-existing column is been dropped, i just commented the dropcolumn part so as for the migration to pass

Reply:
oh!!... I thought it was resolved earlier